plugins/woocommerce-blocks/tests/e2e/tests/catalog-sorting/catalog-sorting.block_theme.spec.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { expect, test } from '@woocommerce/e2e-playwright-utils';
+
+const blockData = {
+	name: 'Catalog Sorting',
+	slug: 'woocommerce/catalog-sorting',
+	class: '.wc-block-catalog-sorting',
+};
+
+test.describe( `${ blockData.slug } Block`, () => {
+	test( "block can't be inserted in Post Editor", async ( {
+		editorUtils,
+		admin,
+	} ) => {
+		await admin.createNewPost();
+		await expect(
+			editorUtils.insertBlockUsingGlobalInserter( blockData.name )
+		).rejects.toThrow();
```

Comment:
What do you think if we use the `insertBlock` utility instead? It throws immediately so we won't need to wait for the full locator timeout, for example:

```suggestion
		await expect(
			editor.insertBlock( { name: blockData.slug } )
		).rejects.toThrow(
			new RegExp( `Block type '${ blockData.slug }' is not registered.` )
		);
```

Reply:
Good suggestion! I addressed your feedback! 🙇